rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -84,8 +84,9 @@ def _value(self, obs):
         return self.model.value_function()[0]
 
 
-A3CTorchPolicy = build_torch_policy(
+A3CTorchPolicy = build_policy_class(
```

Comment:
Can we do these kind of unrelated changes in a separate PR?

Reply:
done

Reply:
Also, review this one here first before proceeding: https://github.com/ray-project/ray/pull/13091